core/src/banking_stage/consumer.rs

```diff
@@ -545,6 +559,18 @@ impl Consumer {
         });
         execute_and_commit_timings.collect_balances_us = collect_balances_us;
 
+        let min_max = batch
+            .sanitized_transactions()
+            .iter()
+            .filter_map(|transaction| {
+                let message = transaction.message();
+                process_compute_budget_instructions(message.program_instructions_iter()).ok()
```

Comment:
Perhaps better to call `get_transaction_priority_details(transaction)` instead of invoking `process_compute_budget_instructions()` directly. This would make refactoring out trait `GetTransactionPriorityDetails` later a bit easier.